app/assets/stylesheets/components/_container.sass

```diff
@@ -1,4 +1,4 @@
-$Container-maxWidth: 960px
+$Container-maxWidth: 1440px
```

Comment:
960px is the usual grid system. 1440px is too much, if you use it on tablets it just breaks. @azevedo-252  @fcbaila